packages/admin/admin/src/EditDialog.tsx

```diff
@@ -87,7 +89,15 @@ interface IHookProps {
     api: IEditDialogApi;
 }
 
-const EditDialogInner: React.FunctionComponent<IProps & IHookProps> = ({ selection, selectionApi, api, title: maybeTitle, children }) => {
+const EditDialogInner: React.FunctionComponent<IProps & IHookProps> = ({
+    selection,
+    selectionApi,
+    api,
+    title: maybeTitle,
+    disableCloseAfterSave = false,
+    onAfterSave: passedOnAfterSave,
```

Comment:
Now that there's no inner onAfterSave anymore, you could simply call it `onAfterSave`